src/library/analysisfeature.cpp

```diff
@@ -156,7 +156,7 @@ void AnalysisFeature::slotAnalyzerQueueProgress(
 }
 
 void AnalysisFeature::slotAnalyzerQueueEmpty(int /*finishedCount*/) {
-    slotAnalyzerQueueDone();
+    m_pAnalyzerQueue->cancel();
```

Comment:
Warning! If I am following the code correctly, this might break the PlayerManager analysis, since it will destroy the queue once it finishes scanning the track, and in PlayerManager the Queue is not recreated.

Reply:
PlayerManager has it's own AnalyzerQueue that is never abandoned.